static/sass/_global-settings.scss

```diff
@@ -16,4 +16,4 @@ $font-display-option: fallback;
 $breakpoint-navigation-threshold: 1250px;
 $threshold-6-12-col: $breakpoint-navigation-threshold;
 $theme-default-nav: dark;
-$grid-max-width: 1440px;
+$grid-max-width: 80rem;
```

Comment:
According to a converter I used `1440px` is `90rem`. Was the purpose of this change to reduce the `grid-max-width `or just to convert it to `rem`